docs/pages/eas-update/debug-advanced.mdx

```diff
@@ -1,19 +1,39 @@
 ---
-title: Debug EAS Update
-sidebar_title: Debug
-description: Learn how to debug EAS Update.
+title: Advanced EAS Update Debugging
+sidebar_title: Advanced
+description: Learn advanced strategies on how to debug EAS Update.
 ---
 
 import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 import { Terminal } from '~/ui/components/Snippet';
 
-It's important to tell the current state of our app at any given time. We built EAS Update with this in mind. Once we know which updates are running on which builds, we can make changes so that our apps are in the state we expect and desire. This guide sets out to show how we can verify our EAS Update and expo-updates configuration so that we can find the source of problems like an app not showing a published update.
+After verifying our EAS Update configuration in our [basic guide](/eas-update/debug), we can move on to more advanced debugging strategies. The following sections describe common classes of problems and strategies on how to tackle them.
 
-## expo-updates configuration
+## General strategies
 
-The expo-updates library runs inside an end-user's app and makes requests to an update server to get the latest update.
+Try these strategies before using the more specific ones mentioned in this guide.
 
-### Verifying app configuration
+### Use `expo-dev-client`
+
+Create a [development version of our build](/eas-update/expo-dev-client). It will help us preview published updates inside a problematic build.
+
+### In-app debugging
+
+The `expo-updates` library exports a variety of functions to interact with updates once the app is already running. In certain cases, making a call to fetch an update and seeing an error message can help us narrow down the root cause. We can make a simulator build of the project and manually check to see if updates are available or if there are errors when fetching updates.
+
+- Print the [Update.Constants](/versions/latest/sdk/updates/#constants) to verify our configuration.
+- [Examine log entries](/versions/latest/sdk/updates/#updatesreadlogentriesasyncmaxage) surfaced from the native layer.
+- Fetch and [load updates manually](/versions/latest/sdk/updates/#check-for-updates-manually).
+
+## Configuration issues
+
+Our app is still not receiving the expected update despite following the [basic guide](/eas-update/debug).
+
+### `expo-updates` configuration
+
+The `expo-updates` library runs inside an end-user's app and makes requests to an update server to get the latest update.
+
+#### Verifying app configuration
```

Comment:
Part of me wonders if this section on verifying general app configuration should go in the basic guide. wdyt @kbrandwijk

Reply:
actually, nm. There are people who use EAS Update but not EAS Build, as shown by this report here: https://exponent-internal.slack.com/archives/C013ZK4SA12/p1687966348035039 . And the basic guide relies on users to be making builds with EAS